x/rewards/spec/README.md

```diff
@@ -0,0 +1,83 @@
+<!--
+order: 0
+title: Rewards Overview
+parent:
+  title: "rewards"
+-->
+
+# `Rewards`
+
+## Overview
+
+The module enables Cosmos SDK-based blockchain to calculate and distribute dApp rewards within the Archway protocol.
+Module also introduces a concept of *minimal consensus fee* to set the lower bound of a transaction fee.
+
+### dApp rewards
+
+Contract rewards are estimated based on a particular contract usage within one block using gas tracking data from the `x/tracking` module.
```

Comment:
Title starts with dApp rewards and proceeds to mention contract rewards, this might be conflicting